### DISCUSSION TITLE
GPU Launch and AmrCore tutorials missing?

### DISCUSSION THREAD

**Discussion by ElloiseFangelLloyd** [AMReX-Codes/amrex#4262] [Q&A]
Hello

In the [docs for the gpu example codes](https://amrex-codes.github.io/amrex/tutorials_html/GPU_Tutorial.html#launch), some information is given about two codes - Launch and AmrCore. 
Those codes are not in [the GPU folder](https://github.com/AMReX-Codes/amrex-tutorials/tree/main/ExampleCodes/GPU) in the Example Codes. Is there a reason for this - am I missing something obvious? The CNS example is there, but not the two others.
I am interested in using AMReX with OpenMP offloading, so I would like to use the Launch example, if possible. If anyone knows of other information on the subject please let me know :) 

Thanks in advance!

**Comment by WeiqunZhang** [comment]
There were some file reorganizations in the past, and the documentation was not updated. Nevertheless, you could still find the old code here. https://github.com/AMReX-Codes/amrex/tree/384ca2341889a8f1db78e87cb4b1735bd2d58554/Tutorials/GPU/Launch
